pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go

```diff
@@ -37,7 +37,7 @@ func TestMostAllocatedScoringStrategy(t *testing.T) {
 		{Name: string(v1.ResourceMemory), Weight: 1},
 	}
 	extendedRes := "abc.com/xyz"
-	extendedResourceLeastAllocatedSet := []config.ResourceSpec{
+	extendedResourceMostAllocatedSet := []config.ResourceSpec{
```

Comment:
I find it's independent of the score algorithms and is used both in `TestLeastAllocatedScoringStrategy` and `TestMostAllocatedScoringStrategy` with different names.
Maybe we can define it separately in `test_util.go` and use it elsewhere? WDYT?